service-front/module/Application/view/layout/layout.twig

```diff
@@ -5,6 +5,8 @@
     {%- set defaultTitle = 'Make a lasting power of attorney' -%}
     <title>{% if (authError is not empty or error is not empty or (form is not empty and form.getMessages|length > 0)) %}Error: {% endif %}{% if overridePageTitle %}{{ overridePageTitle }} - {% elseif pageTitle %}{{ pageTitle }} - {% else %}{% set pageTitle = defaultTitle %}{% endif %}{{ defaultTitle }} - GOV.UK</title>
 
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-1DVC295G9L"></script>
```

Comment:
Are we loading this every time? Is tracking disabled correctly when this is always loaded?

Reply:
It is loaded every time, it was disabled from when I tested it before but I might as well conditionally load the script since the controller already checks the usage cookie

Reply:
EDIT: turns out of course that I'd have to set it in the base controller and not just CookiesController

Reply:
Ticket made for this